### ISSUE TITLE
Composed or Composite rule?

### ISSUE THREAD

**Issue by Brynanders** [w3c/wcag-act#243]
Under [ACT Rule types](https://www.w3.org/TR/act-rules-format/#rule-types) rules comprising of multiple atomic rules are refered to as 'Composed' rules. Shouldn't this be 'Composite' instead?

Composed:
Adjective
- 1. made up of several parts or elements.
- 2. relating to or denoting a classical order of architecture consisting of elements of the Ionic and Corinthian orders.

Noun
- a thing made up of several parts or elements.

Verb
- combine (two or more images) to make a single picture.

**Comment by moekraft** [comment]
I agree. _Composed_ in the adjective form means _calm, tranquil, serene_ whereas the verb _compose_ means _to make or form by combining things, parts, or elements:_ Likewise, _composite_ in the adjective form means _made up of disparate or separate parts or elements; compound_.

**Comment by maryjom** [comment]
Meeting notes: Composite it is.

**Comment by moekraft** [comment]
got it. :)

**Comment by maryjom** [comment]
@moekraft I reviewed the changes and they look good to me.
